doc/source/ray-air/examples/pytorch_tabular_batch_prediction.py

```diff
@@ -0,0 +1,36 @@
+import numpy as np
+import torch.nn as nn
+
+import ray
+from ray.data.preprocessors import Concatenator
+from ray.train.torch import to_air_checkpoint, TorchPredictor
+from ray.train.batch_predictor import BatchPredictor
+
+
+def create_model(input_features):
+    return nn.Sequential(
+        nn.Linear(in_features=input_features, out_features=16),
+        nn.ReLU(),
+        nn.Linear(16, 16),
+        nn.ReLU(),
+        nn.Linear(16, 1),
+        nn.Sigmoid(),
+    )
+
+
+dataset = ray.data.read_csv("s3://anonymous@air-example-data/breast_cancer.csv")
+num_features = len(dataset.schema().names) - 1
+dataset = dataset.drop_columns(["target"])
+
+prep = Concatenator(dtype=np.float32)
+
+checkpoint = to_air_checkpoint(model=create_model(num_features), preprocessor=prep)
+# You can also fetch a checkpoint from a Trainer
+# checkpoint = best_result.checkpoint
+
+batch_predictor = BatchPredictor.from_checkpoint(checkpoint, TorchPredictor)
+
+predicted_probabilities = batch_predictor.predict(dataset)
+predicted_probabilities.show()
+# {'predictions': array([1.], dtype=float32)}
+# {'predictions': array([0.], dtype=float32)}
```

Comment:
Should we also show how one should do scoring and get final accuracies from the predictions?

Reply:
I'd like to not block on this. Can you show me how to do this?

I tried to update the example to do this but:
```python

predicted_probabilities = batch_predictor.predict(
    dataset, feature_columns=features, keep_columns=["target"])
predicted_probabilities.show()
# {'predictions': array([1.], dtype=float32)}
# {'predictions': array([0.], dtype=float32)}

# Calculate final accuracy.
def calculate_accuracy(df):
   return pd.DataFrame({"correct": df["predictions"] == df["target"]})

correct = predicted_probabilities.map_batches(calculate_accuracy)
print("Final accuracy: ", correct.sum(on="correct") / correct.count())
```

But i'm getting:
```python
(map pid=91224)   File "pyarrow/_compute.pyx", line 327, in pyarrow._compute.Function.call
(map pid=91224)   File "pyarrow/error.pxi", line 143, in pyarrow.lib.pyarrow_internal_check_status
(map pid=91224)   File "pyarrow/error.pxi", line 120, in pyarrow.lib.check_status
(map pid=91224) pyarrow.lib.ArrowNotImplementedError: Function sum has no kernel matching input types (array[extension<arrow.py_extension_type<ArrowTensorType>>])
```

Reply:
Made a new PR https://github.com/ray-project/ray/pull/26636/ which adds this section.

The changes are here https://github.com/ray-project/ray/pull/26636/commits/85b7b088fb7a487768dacf0f02107452d11de0d8.

We can either merge that into this one, or if you would like to not block on it, then we can merge that in as a follow up.